<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>376</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>376</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>432</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculatrice</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>311</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="inputHorizontalLayout">
     <item>
      <widget class="QLineEdit" name="InputLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ReturnPushButton">
       <property name="text">
        <string>RETURN</string>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>320</width>
      <height>170</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="KeyBoardGridLayout">
     <item row="1" column="0">
      <widget class="QPushButton" name="SevenPushButton">
       <property name="text">
        <string>7</string>
       </property>
       <property name="shortcut">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="FourPushButton">
       <property name="text">
        <string>4</string>
       </property>
       <property name="shortcut">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="OnePushButton">
       <property name="text">
        <string>1</string>
       </property>
       <property name="shortcut">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="EightPushButton">
       <property name="text">
        <string>8</string>
       </property>
       <property name="shortcut">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="FivePushButton">
       <property name="text">
        <string>5</string>
       </property>
       <property name="shortcut">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="TwoPushButton">
       <property name="text">
        <string>2</string>
       </property>
       <property name="shortcut">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="ZeroPushButton">
       <property name="text">
        <string>0</string>
       </property>
       <property name="shortcut">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="NinePushButton">
       <property name="text">
        <string>9</string>
       </property>
       <property name="shortcut">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="SixPushButton">
       <property name="text">
        <string>6</string>
       </property>
       <property name="shortcut">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="ThreePushButton">
       <property name="text">
        <string>3</string>
       </property>
       <property name="shortcut">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="DividePushButton">
       <property name="text">
        <string>/</string>
       </property>
       <property name="shortcut">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="MultiplyPushButton">
       <property name="text">
        <string>*</string>
       </property>
       <property name="shortcut">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="SubstractPushButton">
       <property name="text">
        <string>-</string>
       </property>
       <property name="shortcut">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="AddPushButton">
       <property name="text">
        <string>+</string>
       </property>
       <property name="shortcut">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="ExpressionPushButton">
       <property name="text">
        <string>'</string>
       </property>
       <property name="shortcut">
        <string>'</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="ConstantTypeDelimiterPushButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="ComplexDelimiterPushButton">
       <property name="text">
        <string>$</string>
       </property>
       <property name="shortcut">
        <string>$</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="EspacePushButton">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>SPACE</string>
       </property>
       <property name="shortcut">
        <string>Space</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="DeletePushButton">
       <property name="text">
        <string>CLEAR</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>160</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="ParametresGridLayout">
     <item row="1" column="0">
      <widget class="QGroupBox" name="ComplexesGroupBox">
       <property name="title">
        <string>Complexes</string>
       </property>
       <widget class="QRadioButton" name="ComplexesYesRadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Oui</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="ComplexesNoRadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Non</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="AnglesGroupBox">
       <property name="title">
        <string>Angles</string>
       </property>
       <widget class="QRadioButton" name="AnglesDegreesRadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Degr√©s</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="AnglesRadianRadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Radians</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="ConstantTypeGroupBox">
       <property name="title">
        <string>Type</string>
       </property>
       <widget class="QRadioButton" name="IntegerRadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Entier</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="RationalRadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Rationnel</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="RealRadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>R√©el</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSpinBox" name="StackDisplaySizespinBox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>270</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="StackDisplayTextEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>121</width>
      <height>251</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="tabStopWidth">
     <number>80</number>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>321</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="PowPushButton">
       <property name="text">
        <string>POW</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="SinPushButton">
       <property name="text">
        <string>SIN</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="CosPushButton">
       <property name="text">
        <string>COS</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="TanPushButton">
       <property name="text">
        <string>TAN</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="LnPushButton">
       <property name="text">
        <string>LN</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="LogPushButton">
       <property name="text">
        <string>LOG</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="InvPushButton">
       <property name="text">
        <string>INV</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="SinhPushButton">
       <property name="text">
        <string>SINH</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="CoshPushButton">
       <property name="text">
        <string>COSH</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="TanhPushButton">
       <property name="text">
        <string>TANH</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="ModPushButton">
       <property name="text">
        <string>MOD</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="SignPushButton">
       <property name="text">
        <string>SIGN</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="SqrPushButton">
       <property name="text">
        <string>SQR</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="CubePushButton">
       <property name="text">
        <string>CUBE</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="SqrtPushButton">
       <property name="text">
        <string>SQRT</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>!</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>77</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="PileOpGridLayout">
     <item row="1" column="0">
      <widget class="QPushButton" name="SumPushButton">
       <property name="text">
        <string>SUM</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="MeanPushButton">
       <property name="text">
        <string>MEAN</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="ClearPushButton">
       <property name="text">
        <string>CLEAR</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+C</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="DupPushButton">
       <property name="text">
        <string>DUP</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QPushButton" name="DropPushButton">
       <property name="text">
        <string>DROP</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QPushButton" name="EvalPushButton">
       <property name="text">
        <string>EVAL</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="SwapPushButton">
       <property name="text">
        <string>SWAP</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Affichage:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAffichage">
    <property name="title">
     <string>Affichage</string>
    </property>
    <addaction name="actionClavier"/>
   </widget>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionQuitter"/>
    <addaction name="actionA_propos"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionAnnuler"/>
    <addaction name="actionR_tablir"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
   <addaction name="menuAffichage"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClavier">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Clavier</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="actionAnnuler">
   <property name="text">
    <string>Annuler</string>
   </property>
  </action>
  <action name="actionR_tablir">
   <property name="text">
    <string>R√©tablir</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MainWindow</sender>
   <signal>keyPressed(QKeyEvent*)</signal>
   <receiver>MainWindow</receiver>
   <slot>keyPressEvent(QKeyEvent*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>keyPressed(QKeyEvent*)</signal>
  <slot>keyPressEvent(QKeyEvent*)</slot>
 </slots>
</ui>
